pkg/plugins/models.go

```diff
@@ -29,6 +29,14 @@ type PluginSignatureState struct {
 
 type PluginSignatureStatus string
 
+func (pss PluginSignatureStatus) IsValid() bool {
+	return pss == pluginSignatureValid
+}
+
+func (pss PluginSignatureStatus) IsInternal() bool {
+	return pss == pluginSignatureInternal
+}
+
```

Comment:
I assumed `PluginSignatureStatus` values are not exposed for some reason, so I decided to create these methods to decouple the values itself from the status (valid, invalid, etc). But I'm completely open to any other alternative.